api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/BaseDaoImpl.java

```diff
@@ -598,18 +598,22 @@ protected void handlePatientReference(Criteria criteria, ReferenceAndListParam p
 			handleAndListParam(patientReference, patientToken -> {
 				switch (patientToken.getChain()) {
 					case Patient.SP_IDENTIFIER:
-						criteria.createAlias("p.identifiers", "pi").add(ilike("pi.identifier", patientToken.getValue()));
+						if (!containsAlias(criteria, "pi"))
```

Comment:
I know this isn't syntactically required but OpenMRS prefer curly braces around all if statements [see our coding style guide](https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Conventions). The reason for that is that while things might work well now, it's easier for maintenance if the if blocks are consistent.